src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -281,6 +308,22 @@ public String toString()
                      getStress());
     }
 
+    /**
+     * Gets the "stress" of the bridge, represented as a double between 0 and 1 (though technically the value
+     * can exceed 1).
+     * @return this bridge's stress level
+     */
+    public double getStress()
+    {
+        if (usePacketRateStat)
+        {
+            return getStressFromPacketRate();
+        }
+        // While a stress of 1 indicates a bridge is fully loaded, we allow
+        // larger values to keep sorting correctly.
+        return (lastReportedStressLevel + Math.max(0, getRecentlyAddedEndpointCount()) * AVG_PARTICIPANT_LOAD);
+    }
+
     /**
      * Returns the "stress" of the bridge. The stress is computed based on the
```

Comment:
This comment should be updated to make clear that it's the backward-compatibility mode